src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -126,28 +107,24 @@ def requestRecord(self, wfData):
                 'campaign': []}
 
     def transferRequest(self, req):
-        "Send request to Phedex and return status of request subscription"
+        """
+        Send request to Phedex and return status of request subscription
+        :param req: request object
+        :return: subscriptoin dictionary {"dataset":transferIDs}
+        """
         datasets = req.get('datasets', [])
         sites = req.get('sites', [])
+        sdict = {}
         if datasets and sites:
-            self.logger.debug("### creating subscription for: %s", pformat(req))
-            subscription = PhEDExSubscription(datasets, sites, self.msConfig['group'])
-            # TODO: implement how to get transfer id
-            tid = hashlib.md5(str(subscription)).hexdigest()
+            self.logger.debug(
+                "### creating subscription for: %s", pformat(req))
+            subscription = PhEDExSubscription(
+                datasets, sites, self.msConfig['group'])
+            self.logger.info(
```

Comment:
Breaking all these lines make the code harder to read IMO. In general, up to 80 chars in the same line is fine, if it goes beyond that, we might either break into multiple lines or not, it depends on how readable it is.

Reply:
please be concrete what you want, the code goes above 80 characters, that's why I break it according to pylint suggestion. So either comply with pylint or not but don't ask for both :)